continuous_integration/appveyor/setup_conda_environment.cmd

```diff
@@ -26,7 +26,7 @@ copy NUL %CONDA_PREFIX%\conda-meta\pinned
 @rem Install optional dependencies for tests
 %CONDA_INSTALL% numpy pandas cloudpickle distributed
 %CONDA_INSTALL% bcolz bokeh h5py ipython lz4 psutil pytables s3fs scipy
-%CONDA_INSTALL% -c conda-forge fastparquet snappy
+%CONDA_INSTALL% -c conda-forge fastparquet snappy zarr
```

Comment:
Based on the appveyor logs it looks like we're able to successfully install `zarr` from conda-forge, however the `zarr`-realted tests are being skipped due to not being able to import `zarr` (ref https://ci.appveyor.com/project/daskdev/dask/builds/23862225#L1141). Not sure exactly why this is happening, @jakirkham do you have any thoughts on what could be causing this?

Reply:
I noted from the error logs that some of the dependencies of zarr did not appear to install properly.

Reply:
Opened up #4706 to track and handle such issues

Reply:
Nothing comes to mind. If something does, I'll add it to that issue.

Reply:
Great, thank you for taking a look @jakirkham!